aiohttp/web_reqrep.py

```diff
@@ -240,6 +240,22 @@ def if_modified_since(self, _IF_MODIFIED_SINCE=hdrs.IF_MODIFIED_SINCE):
         return None
 
     @reify
+    def authorization(self, _AUTHORIZATION=hdrs.AUTHORIZATION):
+        """Parse the Authorization header and return the username and password,
+        or ``None`` if the header is absent or cannot be parsed.
+
+        :rtype:  :class:`aiohttp.helpers.BasicAuth`
+
+        """
+        header = self.headers.get(_AUTHORIZATION)
+        if header is not None:
+            try:
+                return BasicAuth.decode(header)
```

Comment:
Here is the trick: 
```
>>> header = 'basic aGVsbG860LzQuNGAIQ=='
>>> auth = BasicAuth.decode(header)
>>> auth.password
'Ð¼Ð¸Ñ\x80!
'>>> bytearray([ord(i) for i in auth.password]).decode('utf-8')
'мир!'
```

So, unfortunately, this method will work correctly until you stay in latin1 name space. If your users will be allowed to have unicode wide password, they'll have to reimplement it. Somehow you'll have to inject encoding here. Thoughts?